plugin-server/src/worker/vm/extensions/storage.ts

```diff
@@ -40,6 +40,32 @@ export function createStorage(server: Hub, pluginConfig: PluginConfig): StorageE
         })
     }
 
+    const updateStatusTime = async function (key: string): Promise<void> {
```

Comment:
non-blocking

If possible I'd just drop this method into something specific to exports state handling. I would have thought you'd get a typing issue here atm fwiw

Reply:
Type checkers were screaming in my Emacs. I _dynamically_ stopped listening to them.

Reply:
But yeah, I'll move this somewhere in export historical events...